packages/tag/tag.scss

```diff
@@ -4,19 +4,26 @@
 $_tag-background-color: jkl.$color-varde;
 $_tag-color: jkl.$color-granitt;
 
+$_dismiss-focus-border-color: jkl.$color-granitt;
+$_dismiss-hover-color: jkl.$color-stein;
```

Comment:
Disse vil jeg tro blir for lite synlige mot mørk bakgrunn.

Reply:
Tags har ikke mørk bakgrunn?

Reply:
Jeg så for meg at denne var utenfor tagen 😅 Carry on.